modules/datahandling/userrequests_test.go

```diff
@@ -250,8 +250,8 @@ func TestUserProjectsRequest_Process(t *testing.T) {
 		t.Fatalf("Process function responded with status: %d", resp.Status)
 	}
 	// is the data actually correct
-	projects = reflect.ValueOf(resp.Data).FieldByName("Projects").Interface().([]dbfs.ProjectMeta)
-	if len(projects) != 2 && projects[0] != genesproject && projects[1] != notgenesproject {
+	projects = reflect.ValueOf(resp.Data).FieldByName("Projects").Interface().([]projectLookupResult)
+	if len(projects) != 2 && projects[0].ProjectID != genesproject.ProjectID && projects[1].ProjectID != notgenesproject.ProjectID {
```

Comment:
should we add a check to this test which grants someone a permission on it so we can see if they show up in the map (as well as the owner)? I think that would be good since the functionality was added.